beaver_api/lib/src/formatter/html_formatter.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:beaver_trigger_handler/beaver_trigger_handler.dart';
 
 import '../formatter.dart';
+import 'package:xml/xml.dart';
```

Comment:
Importing third party library should be before importing own file.

Use WebStorm's "Optimize Imports".

You can find the menu, "Code -> Optimize Imports".